sqlglot/dialects/clickhouse.py

```diff
@@ -1045,6 +1045,20 @@ def combinedparameterizedagg_sql(self, expression: exp.CombinedParameterizedAgg)
         def placeholder_sql(self, expression: exp.Placeholder) -> str:
             return f"{{{expression.name}: {self.sql(expression, 'kind')}}}"
 
+        def subquery_sql(
+            self, expression: exp.Subquery, sep: str = " AS ", wrap_left="(", wrap_right=")"
```

Comment:
Why do we need both `wrap_left` and `wrap_right`? I think a single `wrap` kwarg suffices here.

Reply:
We need to get rid of extra () during transpiling. By default wrap function add () even if there is no alias. By adding this kw args I allow caller control whether or not to add ()

Reply:
You shouldn't have to specify each delimiter separately; my point was that if we went this way, it would be enough to just have a `wrap` kwarg that controls whether you'll use `self.wrap` in `subquery_sql`.

Reply:
I already tried it and it didn't work, because wrap does extra work, not just adding (). When I implemented it this way it broke the tests